Using the library cmake fails to pick up OpenSSL::SSL

Hey, I am trying to write a test project to read out of a simple file in a bucket and when I build the project with cmake I get the following warnings:

```CMake Error at CMakeLists.txt:4 (add_executable):
  Target "sample" links to target "OpenSSL::SSL" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```

Naturally, when I call `make` it fails:

```[ 50%] Linking CXX executable sample
/usr/bin/ld: cannot find -lOpenSSL::SSL
/usr/bin/ld: cannot find -lOpenSSL::Crypto
/usr/bin/ld: cannot find -lZLIB::ZLIB
collect2: error: ld returned 1 exit status
CMakeFiles/sample.dir/build.make:98: recipe for target 'sample' failed
make[2]: *** [sample] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/sample.dir/all' failed
make[1]: *** [CMakeFiles/sample.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

Here is the code I am trying to run:
```#include "google/cloud/storage/client.h"
#include <condition_variable>
#include <fstream>
#include <future>
#include <iomanip>
#include <mutex>
#include <sstream>
#include <vector>

namespace gcs = google::cloud::storage;
int main(int argc, char* argv[]) try {
  std::string const project_id = "proj";
  std::string const bucket = "bucket";
  std::string const object = "test.txt";
  google::cloud::StatusOr<gcs::ClientOptions> opts =
      gcs::ClientOptions::CreateDefaultClientOptions();
  if (!opts) {
    std::cerr << "Couldn't create gcs::ClientOptions, status=" << opts.status();
    return 1;
  }
  gcs::Client client(opts->set_project_id(project_id));
  // The main thread just reads the file one line at a time.
  auto is = client.ReadObject(bucket, object);
  std::string line;
  std::getline(is, line, '\n');
  int lineno = 0;
  std::cout << "# HEADER " << line << "\n";
  std::cout << "Reading input file " << std::flush;
  /*while (!is.eof()) {
    ++lineno;
    std::getline(is, line, '\n');
    if (line.empty()) {
      break;
    }
    std::cout << line << std::flush;
  }*/
  
  std::cout << " DONE\n";
  return 0;
} catch (std::exception const& ex) {
  std::cerr << "Standard exception raised: " << ex.what() << "\n";
  return 1;
```

And this is the CMakeLists.txt:
```project(sample)
cmake_policy(SET CMP0003 NEW)
cmake_policy(SET CMP0028 NEW)
add_executable(sample main.cc)
find_package(google_cloud_cpp_common REQUIRED)
find_package(storage_client REQUIRED)
target_link_libraries(sample storage_client google_cloud_cpp_common)
```

I am using Ubuntu 18.04. Thanks for any help.

--- a/google/cloud/storage/config.cmake.in
+++ b/google/cloud/storage/config.cmake.in
@@ -16,6 +16,8 @@ include(CMakeFindDependencyMacro)
 find_dependency(CURL)
 find_dependency(Crc32c)
 find_dependency(google_cloud_cpp_common)
+find_dependency(OpenSSL)
+find_dependency(ZLIB)
 
 # Some versions of FindCURL do not define CURL::CURL, so we define it ourselves.
 if (NOT TARGET CURL::CURL)

